tests/model_serving/model_runtime/vllm/conftest.py

```diff
@@ -0,0 +1,133 @@
+from typing import Any, Generator
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.serving_runtime import ServingRuntime
+from ocp_resources.inference_service import InferenceService
+
+# from utilities.infra import s3_endpoint_secret
+from ocp_resources.secret import Secret
+from ocp_resources.service_account import ServiceAccount
+from tests.model_serving.model_runtime.vllm.utils import kserve_s3_endpoint_secret
+from utilities.constants import KServeDeploymentType
+from pytest import FixtureRequest
+from syrupy.extensions.json import JSONSnapshotExtension
+from tests.model_serving.model_runtime.vllm.utils import get_runtime_manifest
+from tests.model_serving.model_server.utils import create_isvc
+from tests.model_serving.model_runtime.vllm.constant import TEMPLATE_MAP, ACCELERATOR_IDENTIFIER, PREDICT_RESOURCES
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="class")
+def ci_s3_storage_uri(request: FixtureRequest, models_s3_bucket_name: str) -> str:
+    return f"s3://{models_s3_bucket_name}/{request.param['model-dir']}/"
```

Comment:
I think this same fixture is already defined in `tests/model_serving/model_server/authentication/conftest.py`

Reply:
this secret create is different from what you are referring here i am creating using staring data also the secret here is more for kserve purposes(using annotation) . Also this one will work on disconnected cluster

Reply:
I think you mixed up the comments? this is the URI string, not a secret

Reply:
yeah..sorry ..i have removed this fixture

Reply:
done